custom_types.yaml

```diff
@@ -587,12 +587,12 @@ node_types:
     interfaces:
       Standard:
         configure:
-          implementation: mesos/mesos_master_install.yml
+          implementation: artifacts/mesos/mesos_master_install.yml
           inputs:
-            consul_server_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_host_list_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_peers_ips: { get_attribute: [ HOST, public_address ] }
-            mesos_masters_list_ips: { get_attribute: [ HOST, public_address ] }
+            consul_server_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_host_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_peers_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            mesos_masters_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
```

Comment:
@MatMaul the parameter "public_address" of the nodetype "tosca.nodes.indigo.Compute" is already a list of IPs so it is not needed to add the "[ ]" chars as it will return a list with one item that is a list, that is not the expected behaviour.